src/database/migrations/20240502174600-add_email_templates.js

```diff
@@ -0,0 +1,197 @@
+const moment = require('moment')
+const fs = require('fs')
+const path = require('path')
+const request = require('request')
+const common = require('@constants/common')
+const fileService = require('@services/files')
+
+async function uploadFile(filePath, uploadFilePath) {
+	try {
+		// Check if the file exists
+		if (!fs.existsSync(filePath)) {
+			throw new Error('The file does not exist in the folder: ' + filePath)
+		}
+
+		const uploadFolder = path.dirname(uploadFilePath)
+		const uploadFileName = path.basename(uploadFilePath)
+
+		// Get signed URL for uploading
+		const signedUrlResult = await fileService.getSignedUrl(uploadFileName, '', uploadFolder, true)
+
+		if (!signedUrlResult.result) {
+			throw new Error('FAILED_TO_GENERATE_SIGNED_URL')
+		}
+
+		const fileUploadUrl = signedUrlResult.result.signedUrl
+		const fileData = fs.readFileSync(filePath)
+
+		// Upload file
+		await request({
+			url: fileUploadUrl,
+			method: 'PUT',
+			headers: {
+				'x-ms-blob-type': common.azureBlobType,
+				'Content-Type': 'multipart/form-data',
+			},
+			body: fileData,
+		})
+
+		console.log('File uploaded successfully')
+		console.log('File path: ' + signedUrlResult.result.filePath)
+
+		// Get downloadable URL
+		const downloadableURL = await fileService.getDownloadableUrl(uploadFilePath, true)
+		return downloadableURL.result
+	} catch (error) {
+		console.error('Error uploading file:', error)
+		throw error
+	}
+}
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		try {
+			const defaultOrgId = queryInterface.sequelize.options.defaultOrgId
+
+			if (!defaultOrgId) {
+				throw new Error('Default org ID is undefined. Please make sure it is set in sequelize options.')
+			}
+
+			// Path to the file to be uploaded
+			const fileName = 'emailLogo.png'
+			const filePath = path.join(__dirname, '../../assets', fileName)
+			const uploadFilePath = 'assets/emailLogo.png'
+
+			// Upload file
+			const logoURL = await uploadFile(filePath, uploadFilePath)
+
+			// Your email template data
+			const emailTemplates = [
+				{
+					body: "{{default}}<div><p>Dear {name},</p> The live session scheduled by you - {sessionTitle} begins in 1 hour. Please ensure that you join at least 10 minutes before for the set time to allow Mentees to settle in.</div>{{/default}}{{linkWarning}}<div><p>Please add a meeting link for your scheduled session that starts in less than 1 hour. To add a meeting link, click on the 'edit session' option on the session details page of MentorED.</div></p>{{/linkWarning}}",
+					code: 'mentor_one_hour_before_session_reminder',
+					subject: 'MentorED - Your scheduled session starts in 1 hour',
+				},
+				{
+					body: "<div style='margin:auto;width:100%;max-width:650px;'><p style='text-align:center'><img class='img_path' style='width:200px; max-width:100%; height:auto;' alt='MentorED' src='https://mentoring-dev-storage.s3.ap-south-1.amazonaws.com/email/image/emailLogo.png'></p><div style='text-align:left;'>",
```

Comment:
@nevil-mathew why this "https://mentoring-dev-storage.s3.ap-south-1.amazonaws.com/email/image/emailLogo.png" link added here ?

Reply:
Removed